app/assets/stylesheets/custom.scss

```diff
@@ -75,36 +73,39 @@ a {
 
 .text-listing-index {
   display: inline-block;
-  margin-left: 10px;
-  margin-right: 10px;
+  margin-left: 2px;
   margin-top: 10px;
   overflow: hidden;
-  text-align: center;
   white-space: nowrap;
+  margin-right: 10px;
   width: 90%;
 }
 
 .text-listing-title p {
   display: inline;
-  text-align: center;
+  float: left;
+  margin-right: 10px;
+  width: 100%;
 }
 
-.text-listing-title {
-  float: left;
-  font-size: 20px;
-  margin-bottom: 40px;
+.text-listing-title a {
+  font-size: 18px;
+  margin-bottom: 30px;
   margin-top: 10px;
-  width: 100%;
+  color: #3f3f3f;
+  float: left;
+  margin-right: 30px;
 }
 
 .text-listing-city {
   clear: both;
   padding-bottom: 10px;
+  float: right;
 }
 
 .listing {
   border: 1px solid;
-  background-color: #ffffff;
+  background: #faf8f2;
```

Comment:
Color literals like `#faf8f2` should only be used in variable declarations; they should be referred to via variable everywhere else.